src/Request.cpp

```diff
@@ -92,6 +92,14 @@ auto Request::SetUrl(const std::string& url) -> bool
     return false;
 }
 
+auto Request::GetNumConnects() const -> uint64_t
+{
+    long count;
```

Comment:
I would default the value to 0 in case curl has an error please

Reply:
Updated to set count = 0